cvat/apps/engine/views.py

```diff
@@ -210,15 +216,11 @@ def get_serializer(self, *args, **kwargs):
     def about(request):
         from cvat import __version__ as cvat_version
         about = {
-            "name": "Computer Vision Annotation Tool",
+            "name": settings.ABOUT_INFO["name"],
```

Comment:
Will the response be cached by the browser?

Reply:
Yes, static content will be cached